std_msgs/Header header 
nav_msgs/Odometry[] odometries
geometry_msgs/Point closest_obstacle